### DISCUSSION TITLE
How can i limit data in preload while filtering the data?

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#11498] [Help] [answered]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

SelectFilter::make('assignee')->relationship('assignee', 'first_name')->searchable()->preload(),
on filter i have to preload only first 5 record and other records will be load on searchable.

**Comment by leandrocfe** [comment, marked as answer]
`->optionsLimit(5)` ?

**Reply by zeeshantariq08** [reply]
yes thank you, it worked
